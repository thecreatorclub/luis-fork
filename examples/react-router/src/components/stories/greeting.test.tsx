import * as React from 'react';

import { create } from 'react-test-renderer';

import { Greeting } from '../greeting';

describe('Greeting', () => {
  const Component = () => <Greeting name="Tomas" />;

  it('renders correctly', () => {
    expect(create(<Component />)).toMatchSnapshot();
  });

  it('fails', () => {
    expect('123').toEqual('456');
  });

  return {
    component: Component
  };
});
