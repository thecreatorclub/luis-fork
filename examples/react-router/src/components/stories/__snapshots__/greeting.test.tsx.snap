// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting renders correctly 1`] = `
<div>
  Hello 
  Tomas
</div>
`;
